tests/mean_04.cc

```diff
@@ -0,0 +1,125 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the SampleFlow authors.
```

Comment:
You probably want to correct that:
```suggestion
// Copyright (C) 2020 by the SampleFlow authors.
```